apps/io-services-cms-webapp/src/watchers/on-services-lifecycles-change.ts

```diff
@@ -34,7 +34,11 @@ const onSubmitHandler = (
 const onApproveHandler = (
   item: ServiceLifecycle.ItemType
 ): RequestPublicationAction => ({
-  requestPublication: { id: item.id, data: item.data },
+  requestPublication: {
+    id: item.id,
+    data: item.data,
+    autoPublish: (item.fsm.autoPublish as boolean) ?? false,
```

Comment:
```suggestion
    autoPublish: (item.fsm.autoPublish as boolean) ?? false, // if FSM Lifecycle is in _approved_ state, then `autopublish` will always be set
```

Reply:
if is possibile, try to apply DRY principles....we avoid repeating `autopublish` default value 😉

Reply:
We added a function `getAutoPublishm` in order to handle the default value for `autoPublish` globally.